services/inc/services_dynalib.h

```diff
@@ -114,6 +114,7 @@ DYNALIB_FN(BASE_IDX + 0, services, set_system_error_message, void(const char*, .
 DYNALIB_FN(BASE_IDX + 1, services, clear_system_error_message, void())
 DYNALIB_FN(BASE_IDX + 2, services, get_system_error_message, const char*(int))
 DYNALIB_FN(BASE_IDX + 3, services, jsmn_parse, int(jsmn_parser*, const char*, size_t, jsmntok_t*, unsigned int, void*))
+DYNALIB_FN(BASE_IDX + 4, services, panic_set_hook, void(PanicHook, void*))
```

Comment:
Signature does not match what's in `panic.h` (`const` vs non-`const` `PanicHook`)